pennylane/labs/tests/resource_estimation/templates/test_resource_stateprep.py

```diff
@@ -347,6 +444,36 @@ def test_resource_params(self, state_vector, wires):
 
         assert op.resource_params == {"num_wires": len(wires)}
 
+    @pytest.mark.parametrize(
+        "compact_state, wires, expected_params",
+        (
+            (
+                re.CompactState.from_state_vector(num_qubits=3, num_coeffs=8),
+                range(3),
+                {"num_wires": 3},
+            ),
+            (
+                re.CompactState.from_state_vector(num_qubits=100, num_coeffs=2**100),
+                range(100),
+                {"num_wires": 100},
+            ),
+            (
+                re.CompactState.from_state_vector(num_qubits=3, num_coeffs=8, num_work_wires=5),
+                range(3),
+                {"num_wires": 3},
+            ),
+            (
+                re.CompactState.from_bitstring(num_qubits=7, num_bit_flips=10),
```

Comment:
Both methods `from_state_vector` and `from_bitstring` are supposed to work with `ResourceMottonenStatePreparation`?